setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = localstack
+name = localstack-core
 description = LocalStack - A fully functional local Cloud stack
```

Comment:
Not super critical right now, but I wonder whether we could maybe also reword the description of the distribution to something like "The core library and runtime of LocalStack". Similarly we could probably remove the long description from the distribution, since we have it in the `localstack` distribution.